src/adaptors/adaptor_listener.c

```diff
@@ -136,8 +135,8 @@ static void _listener_event_handler(pn_event_t *e, qd_server_t *qd_server, void
                 }
                 sys_mutex_unlock(&li->lock);
                 if (up)
-                    qd_log(log, QD_LOG_INFO, "Listener %s: listening for client connections on %s", li->name,
-                           li->host_port);
+                    qd_log(log, QD_LOG_INFO, "Listener %s: listening for client connections on %s with backlog %d",
+                           li->name, li->host_port, li->backlog);
```

Comment:
You can keep it one-parameter-per-line if you use the `//` formatting trick shown in https://github.com/skupperproject/skupper-router/blob/main/docs/notes/code-conventions.md#formatting-tips

So maybe next time.